### PR TITLE
1.0 version cleanup

### PR DESCRIPTION
**Pull request by stuwest**
This PR has a smattering of 1.0 version cleanup in documentation plus adding `load_plugin_textdomain()`.

### CODE REVIEW

File: wp-cldr.php [hunk 1]
```diff
@@ -33,20 +33,28 @@ function wp_cldr_menu() {
 }
 
 /**
+ * Loads the text domain for the plug-in. We call this only when the settings page is loaded since translations aren't needed any earlier.
+ */
+function wp_cldr_load_plugin_textdomain() {
+	load_plugin_textdomain( 'wp-cldr', false, basename( dirname( __FILE__ ) ) . '/languages/' );
```
Comment (jblz) [inline wp-cldr.php]: `__DIR__` is equivalent to `dirname( __FILE__ )` & preferable

File: wp-cldr.php [hunk 2]
```diff
@@ -33,20 +33,28 @@ function wp_cldr_menu() {
...
+}
+
+/**
  * Gets the settings and examples page for the plugin.
  */
 function wp_cldr_settings() {
 
+	wp_cldr_load_plugin_textdomain();
```
Comment (jblz) [inline wp-cldr.php]: Do you anticipate calling this anywhere else? If not, I'd say just call `load_plugin_textdomain` here instead.

**Comment by jblz** [comment]
ok to merge once `__DIR__` is used.

File: wp-cldr.php (hunk 2 above)
Comment (stuwest) [inline wp-cldr.php]: No I don't think we'll use it elsewhere. Was just instinctively trying to break things up. Will change.

File: wp-cldr.php (hunk 1 above)
Comment (stuwest) [inline wp-cldr.php]: I blame https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/#loading-text-domain which use the `dirname( __FILE__ )` :)

**Pull request merged**